client/src/extension.ts

```diff
@@ -70,6 +70,11 @@ let client: LanguageClient;
 // });
 
 export function activate(context: ExtensionContext) {
+  let outputChannel = window.createOutputChannel(
+    "ReScript Language Server",
+    "rescript"
+  );
+

```

Comment:
Creating and passing around an explicit output channel lets us have a good place to log things to, that's also easy to reveal programatically.